components/bank/landing.js

```diff
@@ -110,7 +112,7 @@ export default function Landing({ showButton, eventsCount }) {
               fontWeight: 'normal'
             }}
           >
-            Tuscon, AZ
+            Singapore 🇸🇬
```

Comment:
Flag emojis don't work on Windows :(

```suggestion
            Singapore
```